test_libs/pyspec/eth2spec/test/helpers/custody.py

```diff
@@ -19,20 +20,24 @@ def get_valid_early_derived_secret_reveal(spec, state, epoch=None):
             message_epoch=epoch,
         ),
     )
-    mask = bls_sign(
-        message_hash=spec.hash_tree_root(epoch),
+    # Generate the mask (any random 32 bytes will do)
+    mask = reveal[:32]
```

Comment:
```suggestion
    mask = hash(reveal)
```

For the test it does not matter, but in reality it would be a bad idea to use `reveal[:32]` (it may be possible to brute force the full reveal from this, which defeats the purpose of masking). So let's put something here that people can use safely in practice, in case someone happens to use this code as a reference.

Reply:
I had this debate with myself and opted to just use the first 32 bytes for the sake of speed. That said, considering how marginal the cost is, I'm happy to switch over to `hash(x)` for signaling's sake.